docs/Course Wiki/ECE Course Offerings/ECE431.md

```diff
@@ -0,0 +1,75 @@
+
+The midterms are in class (1 hour) and usually not very difficult with the averages being in the high 80's to low/mid 90's with small standard deviations. Very few practice exams are given out (1-3 per exam) and students. Cheatsheet policies may vary depending on the semester and the lead instructor. For the Spring 2023 semester, students were allowed to bring one double-sided 8.5x11 cheatsheet to the first exam, two double-sided 8.5x11 cheat sheets to the second exam, but only two double-sided 8.5x11 cheat sheets to the final exam. Students were highly recommended to reuse cheat sheets from prior exams, although this was not enforced.
+
+Overall, the course requires about 8-12 hours a week based on how difficult the lab is for the given week and also how much effort you put into the report.
+## Instructor
+
+Professor Haran usually teaches this course. Haran has an extensive background within electric machines as he is the head of the power group and oversees many research projects related to electric machinery. He has experience working as an engineer at General Electric. 
+
+## Course Tips
+
+The lab reports can be complicated based on the week and may need extra time to complete, so start early! Especially start the induction machine lab report early as it is one of the longest reports that you will have to write in the course. Professor Haran usually finishes the content early in the class period and expects students to ask questions, so do not be afraid to ask a question no matter how simple you may think it is. 
+The course can get overwhelming quickly as you keep learning about different types of electric machines at an increasing pace. Be sure that you understand the content because the exams (and homeworks) often ask you to explain what would happen if a parameter changed and why that would occur. 
+
+Make sure you fully understand the prelabs before the labs so you do not have to redo your lab experiment and stay after (a common occurrence in this class). Also do not be afraid to ask a TA to double check your wiring before you connect power, you will be working with high voltages and high currents which can potentially harm you if you do not wire the circuit correctly. 
+
+## Life After
+
+This is currently the only ECE class on electric machinery, except for ECE 432 (Advanced Electric Machinery) which has not been taught in over a decade. Related classes include [ECE486 - Control Systems](ECE486.md) (more on control of electric machines), [ECE476 - Power System Analysis](ECE476.md) (focus on the grid with many overlapping concepts), [ECE464 - Power Electronics](ECE464.md) (if you like EVs, inverters, power in general), and [ECE333 - Green Electric Energy](ECE333.md)(more on the grid and green energy).
```

Comment:
ECE 432 -> ECE432. Is that worth linking? Or does that course not exist anymore so we won't write a wiki page for it?